src/components/SubscriptAvatar.js

```diff
@@ -0,0 +1,40 @@
+import React, {memo} from 'react';
+import PropTypes from 'prop-types';
+import {Image, View} from 'react-native';
+import styles from '../styles/styles';
+import Tooltip from './Tooltip';
+
+const propTypes = {
+    /** Array of avatar URL */
+    avatarImageURLs: PropTypes.arrayOf(PropTypes.string).isRequired,
+
+    /** Tooltip for the Avatar */
+    avatarTooltips: PropTypes.arrayOf(PropTypes.string).isRequired,
+};
```

Comment:
We will want to show the `Expensicons.workspace` icon here if  `avatarImageURLs[1] ===''` which will be the case when the users' workspace does not have an avatar set. Without this we'll see this:
<img width="58" alt="image" src="https://user-images.githubusercontent.com/19364431/155604216-77c37b2b-1c9d-42c2-b683-02a505de4ad4.png">


So here let's add a 'defaultSubscriptIcon' prop here that will be of type `PropTypes.func.building.isRequired`. Then we'll conditionally render the icon  if `avatarImageURLs[1] ===''` using `<Icon src ={defaultSubscriptIcon} ... />`. To get the the subscript style you'll have to wrap this in a View and apply the style on that. You'll also have to set the icon's width and height manually.

CC: @marcaaron, @roryabraham if you think this is solid or there is a better way to do this.

Reply:
Then in places where we call <SubscriptAvatar> for the policyExpenseChats we'll set `Expensicons.workspace` as the `defaultSubscriptIcon` prop.

Reply:
What is the significance of `avatarImageURLs[1]`? The user at the 1st index will be an empty string seems a little magical and subject to change unintentionally. Can we maybe just pass a prop that says whether to use this icon or not? Do we have a better way to tell this is a workspace chat and should use this icon?

Reply:
> So here let's add a 'defaultSubscriptIcon' prop here ... [then] conditionally render the icon if avatarImageURLs[1] ==='' using <Icon src ={defaultSubscriptIcon} ... />

I have two suggestions here. First, this `SubscriptAvatar` component can only really have two avatars right? So instead of using an `avatarImageURLs` prop that's an array, where the "main" avatar is at index 0 and the "subscript" avatar is at index 1, why not just have separate props for the `mainAvatar` and the `secondaryAvatar`?

Second, I think we could clean this up further by refactoring the `Avatar` component to make it a bit more reusable. Right now it takes an image URL, and also `isChatRoom` and `isArchivedRoom`, which are used to display static icon assets for chat rooms (which don't have image avatars yet AFAIK). Instead, `Avatar` could more generically take:

1. A source, which can be either an image URL or a static icon
1. A fallback icon, which can only be static icon (available offline).
    
So a simplified version might look something like this:

```js
const propTypes = {
    /** Source for the avatar. Can be the URL of an image or an icon. */
    source: PropTypes.oneOfType([PropTypes.string, PropTypes.func]).isRequired,
    
    /** Icon to render if the source image could not be loaded. */
    fallbackIcon: PropTypes.func,
    
    /** Information about the network */
    network: PropTypes.shape({
        /** Is the network currently offline or not */
        isOffline: PropTypes.bool,
    }),
};

const defaultProps = {
    fallbackIcon: () => {},
    network: {},
};

const Avatar = (props) => {
    if (!props.source || props.network.isOffline) {
        return props.fallbackIcon
            ? <Icon src={props.fallbackIcon} />
            : null;
    }

    return (
        <View style={props.containerStyles}>
            {
                _.isFunction(props.source)
                    ? <Icon src={props.source} />
                    : <Image source={{uri: props.source}} />
            }
        </View>
    );
};
```

I think that could simplify things here and elsewhere.